packages/core/src/application.ts

```diff
@@ -147,6 +155,10 @@ export class Application extends Context {
       const options = OPENAPI_SPEC_MAPPING[request.url];
       return this._serveOpenApiSpec(request, response, options);
     }
+    if (request.method === 'GET' && request.url &&
+        request.url.match(/\/swagger.json\?ui=true/)) {
```

Comment:
I find it weird that a query parameter is changing the response so dramatically. Can we use a plain url path like `/explorer` or `/swagger-ui` instead?

Reply:
That makes sense to me. I will change it to `/swagger-ui`.